### PR TITLE
Breaking Change: Schema\Builder::getAllTables was returning wrong type

### PR DESCRIPTION
**Pull request by taka-oyama**
was list of string but should be row with name and type (differs for various databases)

### CODE REVIEW

File: src/Schema/Builder.php [hunk 1]
```diff
@@ -35,19 +35,13 @@ class Builder extends BaseBuilder
 
 
     /**
-     * @return list<string>
+     * @return list<array{ name: string, type: string }>
```
Comment (tyabu12) [inline src/Schema/Builder.php:38]: This is a breaking change. Could you add this to the CHANGELOG?

File: tests/Schema/BuilderTest.php [hunk 2]
```diff
@@ -197,6 +197,11 @@ public function test_getAllTables(): void
             $table->primary('id');
         });
 
-        $this->assertContains(self::TABLE_NAME_CREATED, $sb->getAllTables());
+        $tables = array_map(
+            static fn (array $row): string => $row['name'],
```
Comment (tyabu12) [inline tests/Schema/BuilderTest.php:201]: How about adding an assertion about `$row['type']`?

**Review by tyabu12 — CHANGES_REQUESTED** [review]

File: src/Schema/Builder.php (hunk 1 above)
Comment (taka-oyama) [reply src/Schema/Builder.php:38]: added

File: tests/Schema/BuilderTest.php (hunk 2 above)
Comment (taka-oyama) [reply tests/Schema/BuilderTest.php]: They have been added.

**Review by tyabu12 — APPROVED** [review]
LGTM

File: src/Schema/Builder.php [hunk 3]
```diff
@@ -35,19 +35,13 @@ class Builder extends BaseBuilder
...
-        /** @var list<array{ TABLE_NAME: string }> $results */
-        $results = $this->connection->select(
+        return $this->connection->select(
             $this->grammar->compileGetAllTables()
         );
-
-        /** @var Processor $processor */
-        $processor = $this->connection->getPostProcessor();
-
-        return $processor->processGetAllTables($results);
```
Comment (halnique) [inline src/Schema/Builder.php:50]: https://github.com/colopl/laravel-spanner/blob/0a9ca9811486dfd45e5eabb2a5c72edd37c2e704/src/Query/Processor.php

Would you also remove `Processor::processGetAllTables`?
(It's a breaking change anyway...)

Comment (taka-oyama) [reply src/Schema/Builder.php:50]: fixed

**Review by zeriyoshi — APPROVED** [review]
LGTM

**Review by tyabu12 — APPROVED** [review]
LGTM

**Pull request merged**